{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name_alphanumeric_underscore}}/training/README.md

```diff
@@ -72,10 +72,10 @@ Otherwise, e.g. if iterating on ML code for a new project, follow the steps belo
   `dbx sync repo --profile {{cookiecutter.project_name}}-dev --source . --dest-repo your-repo-name`, where `your-repo-name` should be the last segment of the full repo name (`/Repos/username/your-repo-name`)
 
 #### Running code on Databricks
```

Comment:
This whole file is talking about mlflow recipes, which is not the default option any more. I wonder if we should just provide this file when recipes is selected, and have a simpler README otherwise.

Reply:
We only return this file when the recipes option is selected and otherwise we don't have a README since it's only a single notebook for the default and the feature store options. This is the same behavior we had for the feature store option prior to this change